include/LightGBM/cuda/vector_cudahost.h

```diff
@@ -7,7 +7,7 @@
 
 #include <LightGBM/utils/common.h>
 
-#ifdef USE_CUDA
+#if defined(USE_CUDA) || defined(USE_CUDA_EXP)
```

Comment:
Update Copyright info for this file
https://github.com/microsoft/LightGBM/blob/3032b646ad8fb31b391e3dbed425e16c4eb400f5/include/LightGBM/cuda/vector_cudahost.h#L2
```
  * Copyright (c) 2020-2011 IBM Corporation, Microsoft Corporation. All rights reserved. 
```

Reply:
Resolved via f44b881

Reply:
Sorry for missing this. Now resolved via 9135582